### PR TITLE
Added a named selects property on selects

### PR DESCRIPTION
**Pull request by nolamesa**

### CODE REVIEW

**Review by tobymao — APPROVED** [review]

File: sqlglot/expressions.py [hunk 1]
```diff
@@ -1133,6 +1133,10 @@ def ctas(self, table, properties=None, dialect=None, parser_opts=None, copy=True
             ),
         )
 
+    @property
+    def named_selects(self):
```
Comment (barakalon) [inline sqlglot/expressions.py:1137]: Is it strange for this to be on the base Expression instance? Should this be on Select?

**Review by barakalon — APPROVED** [review]
Small comment about where to put the method.

Otherwise, sweet! I need this immediately haha

File: sqlglot/expressions.py (hunk 1 above)
Comment (tobymao) [reply sqlglot/expressions.py:1137]: oh, i didn't catch that. yea that doesn't make sense

Comment (tobymao) [reply sqlglot/expressions.py:1137]: what about subqueryable?

**Review by eakmanrq — APPROVED** [review]

File: sqlglot/expressions.py (hunk 1 above)
Comment (nolamesa) [reply sqlglot/expressions.py:1137]: it's on select class

Comment (barakalon) [reply sqlglot/expressions.py:1144]: doh

**Review by barakalon — APPROVED** [review]

**Review by tobymao — APPROVED** [review]

**Pull request merged**